app/mailers/candidate_mailer.rb

```diff
@@ -213,6 +213,16 @@ def changed_offer(application_choice)
     )
   end
 
+  def deferred_offer(application_choice)
+    @application_choice = application_choice
+    @course_option = @application_choice.course_option
```

Comment:
@

> Looks very good to me! Minor qs inline. & maybe one for @adamsilver... should we change this 👇 ? I found it a bit surprising having just deferred the offer.
> 
> <img alt="Screenshot 2020-09-10 at 16 39 08" width="686" src="https://user-images.githubusercontent.com/642279/92756066-5d040a80-f384-11ea-9c0b-12b9039bd762.png">

@duncanjbrown I‘m not sure what exactly you're suggesting we change this to?

I think it's okay as it as, and then when the new cycle comes around it would say ‘Previous cycle’.

Then when the offer has been reconfirmed for the current cycle it should say ‘Current cycle’.

Part of the rationale is that's the way you know whether that deferred offer can be reconfirmed or not.

That is...

If the offer was deferred in cycle 1 and it's now cycle 2 it can be reconfirmed.

If the offered was deferred in cycle 1, and cycle 1 is still in progress then it cannot be reconfirmed.

Reply:
OK, got it! Happy to leave it